src/components/GiveHelp.js

```diff
@@ -1,17 +1,19 @@
 import React from "react"
-
-// Gatsby
 import { StaticQuery, graphql } from "gatsby"
-
-// Styles
 import { StyledGrid } from "../styles/StyledGrid"
 
+import { Elements } from "@stripe/react-stripe-js"
+import { loadStripe } from "@stripe/stripe-js"
+import StripeForm from "./StripeForm"
+import { StyledStripeForm } from "../styles/StyledStripeForm"
+
+const stripePromise = loadStripe("pk_test_kfhmKZBuex2hxS8FQ6t0xoTe003Xv8kQzG")
```

Comment:
@sethburtonhall - Is this value better stored as an environment variable?

Reply:
Yes, will do that.